InternetHospitalWebUI/src/app/Components/Layout/header/header.component.html

```diff
@@ -13,24 +13,31 @@
     <span class="example-fill-remaining-space"></span>
 
     <!-- Phone Menu -->
-    <button mat-icon-button [matMenuTriggerFor]="menu" fxHide.gt-xs>
-        <mat-icon>more_vert</mat-icon>
+    <button mat-icon-button [matMenuTriggerFor]="menu" fxHide.sm fxHide.md fxHide.lg>
+        <mat-icon>arrow_drop_down</mat-icon>
     </button>
     <mat-menu #menu="matMenu">
+        <a mat-button routerLink="/doctor-list">Doctor's List</a>
```

Comment:
i thought that eugen and serhii replaced link to button for better visualization, it`s better to hear their opinion and maybe change 'a' to 'button'